db/seeds.rb

```diff
@@ -5,3 +5,43 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+# Create some users
+User.create!([
```

Comment:
**Kudos for adding preloaded data to your project!** 🏆

Reply:
Thanks for your feedback